### DISCUSSION TITLE
[Unstable] Alien reactor and detectors are shifted

### DISCUSSION THREAD

**Discussion by Minedeployder** [FakeFishGames/Barotrauma#14633] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![1](https://github.com/user-attachments/assets/efb2f484-2e8b-48e5-a98d-6942399a7378)
![2](https://github.com/user-attachments/assets/977a3f19-9e96-4965-8af7-896dd13c2ef0)

As you can see it is placed not where to be expected(stable vs unstable). 
p.s. Can you please make those items "legacy"? Reactor has absolutely different style.

### Reproduction steps

1. Find anything in workshop that contains alien reactor
2. Go into unstable and check where it is placed now

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in our private repo now (by keeping the old versions of the devices as legacy options, and the devices in existing subs/ruins unchanged). Fix coming up in the next unstable build.
